### PR TITLE
Chore/events migrations

### PR DESCRIPTION
**Pull request by JohanHjelsethStorstad**
This PR should be merged #324. It migrates the part of the events system already implemented in #324

### CODE REVIEW

**Comment by JohanHjelsethStorstad** [comment]
@Paulijuz - You want to try?

File: src/prisma/schema/event.prisma [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  eventEnd           DateTime
+  canBeViewdBy       EventCanView
+  hostedByCommitee   Committee?   @relation(fields: [hostedByCommiteeId], references: [id])
+  hostedByCommiteeId Int?
+
+  takesRegistration Boolean
+  places            Int                 @default(0)
+  registrationStart DateTime
+  registrationEnd   DateTime
+  EventRegistration EventRegistration[]
```
Comment (Paulijuz) [inline src/prisma/schema/event.prisma:27]: Not a big think, but it annoys me that the prisma extension PascalCase for automatically added relations when it should be camelCase

**Review by Paulijuz — APPROVED** [review]
LGTM!

Only had minor code style complaint comment, which you can ignore.

Also, had an issue for ombul migrations, i think one of the ombul files might have been changed since the migration db was created? But this has nothing to do with this PR so it can be fixed later.

**Review by Paulijuz — APPROVED** [review]
Awsome!

**Pull request merged**